app/src/debug/google-services.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
Good point. No damage done because it's debug file (however I'm going to remove it from version control), but it's recommended to not commit Google's json files. It's strictly prohibited to not publish server's json config file, if you have your backend's code open sourced.